### PR TITLE
feat: add jest and sample test

### PR DESCRIPTION
**Pull request by ccuilla**
- [ ] Add cucumber setup
- [x] Add sample mocks for some common things like Sentry, cross-fetch
- [x] Fix error:

`Requires Babel "^7.0.0-0", but was loaded with "6.26.3". If you are sure you have a compatible version of @babel/core, it is likely that something in your build process is loading the wrong version. Inspect the stack trace of this error to look for the first entry that doesn't mention "@babel/core" or "babel-core" to see what is calling Babel.`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,3 +33,16 @@ Mongo will be available at: `mongodb://127.0.0.1:27017`
...
 
 `> mongo`
+
+## TODO
+
+- `mkdir utilities` - place for utility code
+- `mkdir packages`
+- `mkdir packages/okgrow` - place for local "pre" npm package-able code
+- `yarn add lodash`
+- `yarn add winston` - for logging
```
Comment (cfnelson) [inline README.md]: Are we using yarn or `npm`? as the script is using `npm` and `npx`?

If this is eventually going to be turned into a npm package, we should stick to using `npm` so we don't have issues with automating code publishing etc.

Also npm 6 has `npm audit` which I think is an awesome tool for checking/fixing security issues in specific pkgs etc. I think we used it a few times on city & stay 🙌

**Comment by cfnelson** [comment]
@ccuilla Guessing I was tagged on this due to the `babel` bug? Was looking at the `pkg.json`. Wondered if the script should declare a `engine` so you can alert someone if they are running this with the wrong npm/node versions.

In regards to `babel` and the `pkg.json`, did the stack trace alert you to which pkg is still using v6 of babel ? I also looked quickly at the webpack config where babel-loader is being used and noticed you weren't passing the options (preset/plugins) and I saw `@babel/plugin-proposal-object-rest-spread` had been added. Not 100% sure if babel-loader will use the plugins if they are not added? 🤷‍♂️ 

**Potential Solution:**
Also if I had to take a **wild guess** I believe you might be able to get this working if you don't know the pkg in question that is causing you problems by adding `"babel-core": "^7.0.0-bridge.0",` to your devDeps and run `npm install`. 

It's what we did in city & stay, it just bridges the name/scope changes due to babel moving to `@babel`. More details on what that pkg is doing here -> https://github.com/babel/babel-bridge

If that doesn't work then, happy to pair with you at some point 😄 , Also happy to have a walk through of what you are doing so I can provide more informed feedback/suggestions.

**Tangent:**
Also potentially you could be using https://github.com/standard-things/esm for providing `ESM` to make life simple if your just wishing to run certain node scripts. 

I mentioned it also in dev due to the performance gains you can get on serverless env's this specific issue discusses aws lambda usage  and you might find interesting https://github.com/standard-things/esm/issues/586
tweet -> https://twitter.com/jdalton/status/1037391541210296320

**Comment by cfnelson** [comment]
Took a quick look at the repo and have to say nice work and documentation so far! Looks nice and neat 🙌

File: README.md (hunk 1 above)
Comment (ccuilla) [reply README.md]: I've been using `yarn`.

**Comment by ccuilla** [comment]
@cfnelson Thanks for the suggestion on trying `"babel-core": "^7.0.0-bridge.0"` but that did not work. We might need to pair on this. I don't have the stack trace handy but its easy to re-produce.

> I also looked quickly at the webpack config where babel-loader is being used and noticed you weren't passing the options (preset/plugins) and I saw @babel/plugin-proposal-object-rest-spread had been added. Not 100% sure if babel-loader will use the plugins if they are not added?

Not sure exactly what you mean here. I pretty much just copied this webpack from this tut: https://hackernoon.com/create-a-serverless-graphql-server-using-express-apollo-server-and-aws-lambda-c3850a2092b5 I did add the GQL loader but nothing else.

My theory is that this issue has something to do with web pack...but cannot be certain.

**Comment by cfnelson** [comment]
@ccuilla No worries, will set aside time this thursday/friday for you. Hit me up tomorrow whenever you like.  Thanks for the link, I gave it a quick read along with the other article about SSR react app using express. 

It's not clear to me if this setup is only for our GraphQL server or will this also handling our SSR React app? Not sure/clear to me if the whole thing will be in the same project or we have two separate projects which we can deploy independently?

If this is only doing graphql etc. then babel might be able to be avoided. Can use some other pkgs like `esm` for providing module support. Anyways lets chat tomz and see if we can solve this babel bug 🙌

**Pull request merged**